webapp/src/components/App/Singlestudy/explore/Results/ResultDetails/index.tsx

```diff
@@ -65,6 +65,7 @@ import ResultFilters from "./ResultFilters.tsx";
 import { toError } from "../../../../../../utils/fnUtils.ts";
 import EmptyView from "../../../../../common/page/SimpleContent.tsx";
 import { getStudyMatrixIndex } from "../../../../../../services/api/matrix.ts";
+import { MatrixGridSynthesis } from "@/components/common/Matrix/components/MatrixGridSynthesis/index.tsx";
```

Comment:
It's normal `index.tsx` at the end?

Reply:
hum it should not appear.. dont know why the auto import added this I will check